app/src/main/res/values/strings.xml

```diff
@@ -147,4 +147,19 @@ OATH Version: %2$s</p>
     <string name="external_qr_scanner_missing">No QR code reader available.</string>
     <string name="toggle_mask">Show/Hide password</string>
 
+    <string-array name="icon_choices">
+        <item>Select an image…</item>
+        <item>Use default icon</item>
+    </string-array>
+
+    <string-array name="defaultKeyboard">
+        <item>US</item>
+    </string-array>
+
+    <string-array name="themeSelection">
+        <item>Light</item>
+        <item>Dark</item>
+        <item>AMOLED</item>
+    </string-array>
+
```

Comment:
I think we can exclude some of string from localization and move the for example to no_translated.xml. For example, app name? copyright text?

Reply:
Good point, but I think even those would require translations, no? The app name "Yubico Authenticator" would be "Autenticador Yubico" in Portuguese, for example. But "AMOLED" is not translatable I think.

Reply:
I moved these back to the arrays file, and will move the strings themselves to the strings xml. For the strings we do not need translations we can add a property on the XML tag.